apiserver/routers/routers.go

```diff
@@ -251,19 +251,29 @@ func NewAPIRouter(han *controllers.APIController, authMiddleware, initMiddleware
 	/////////////////////
 	// Get pool
 	apiRouter.Handle("/repositories/{repoID}/pools/{poolID}/", http.HandlerFunc(han.GetRepoPoolHandler)).Methods("GET", "OPTIONS")
+	apiRouter.Handle("/repositories/{owner}/{repo}/pools/{poolID}/", http.HandlerFunc(han.GetRepoByNamePoolHandler)).Methods("GET", "OPTIONS")
```

Comment:
You probably need to add routes for CRUD for repositories as well, not just for repo pools. Same for repo instances.

Reply:
That's ok. It's what reviews and tests are for.